### PR TITLE
Work on CI

### PR DESCRIPTION
**Pull request by Rombur**
DO NOT MERGE
This is not ready but I need to create PR to test the CI.

### CODE REVIEW

File: .jenkins [hunk 1]
```diff
@@ -0,0 +1,35 @@
+pipeline {
```
Comment (dalg24) [inline .jenkins:1]: Prefer `.jenkins/continuous.groovy`

**Comment by Rombur** [comment]
@janciesko  This ready for review. The fix from #35 is included

**Comment by Rombur** [comment]
@janciesko _ping_

**Review by janciesko — APPROVED** [review]
LGTM

**Pull request merged**